### PR TITLE
Removed admin-bundle dependency

### PR DESCRIPTION
**Pull request by core23**
## Subject

I am targeting this branch, because this contains BC breaks.

## Changelog

```markdown

### Changed
- Replace admin `FormMapper` with internal component

### Removed
- Removed `admin-bundle` dependency

```

### CODE REVIEW

**Comment by kunicmarko20** [comment]
maybe port the test from admin bundle?

**Comment by kunicmarko20** [comment]
(if there is any)

File: tests/Block/Service/RssBlockServiceTest.php [hunk 1]
```diff
@@ -39,7 +40,7 @@ public function testService(): void
 
         $blockContext = new BlockContext($block, $optionResolver->resolve());
 
-        $formMapper = $this->getMockBuilder('Sonata\\AdminBundle\\Form\\FormMapper')
+        $formMapper = $this->getMockBuilder(FormMapper::class)
```
Comment (OskarStark) [inline tests/Block/Service/RssBlockServiceTest.php]: Can we use createMock here?

File: composer.json [hunk 2]
```diff
@@ -35,6 +35,7 @@
         "symfony/http-foundation": "^2.8 || ^3.2 || ^4.0",
         "symfony/http-kernel": "^2.8 || ^3.2 || ^4.0",
         "symfony/options-resolver": "^2.8 || ^3.2 || ^4.0",
+        "symfony/templating": "^2.8 || ^3.2 || ^4.0",
```
Comment (kunicmarko20) [inline composer.json]: this is deprecated, we should probably look into dropping it

**Pull request merged**